lambda_function.py

```diff
@@ -0,0 +1,85 @@
+import os
+import boto3
+import logging
+from botocore.exceptions import ClientError
+
+ses_client = boto3.client('ses')
+
+def lambda_handler(event, context):
+    file_urls = []
+
+    prod_email = os.environ['PROD_EMAIL']
+    staging_email = os.environ['STAGING_EMAIL']
+    sandbox_email = os.environ['SANDBOX_EMAIL']
+
+    prod_bucket = os.environ['PROD_BUCKET']
+    staging_bucket = os.environ['STAGING_BUCKET']
+    sandbox_bucket = os.environ['SANDBOX_BUCKET']
+
+    prod_url = os.environ['PROD_URL']
+    staging_url = os.environ['STAGING_URL']
+    sandbox_url = os.environ['SANDBOX_URL']
+
+    email_recipient = os.environ['EMAIL_RECIPIENT']
+
+    for record in event['Records']:
+        bucket = record['s3']['bucket']['name']
+        key = record['s3']['object']['key']
+
+        if "prod" in bucket:
```

Comment:
Now that this is environment variables, do we need the if statement at all?  Could we not just replace the bucket with the url regardless of the environment?